api-reference/50 Common/Object Structures/ExportDataGridProps/topLeftCell.md

```diff
@@ -9,8 +9,8 @@ inheritsType: CellAddress
 A cell used as a start position for export.
 
 ---
-The cell is specified using coordinates in the Excel document. For example, the default value:
+The cell is specified using coordinates in the Excel document. For example, the default value specifies cell A1:
 
     { row: 1, column: 1 }
 
-... specifies cell A1. 
\ No newline at end of file
+You can also specify the **topLeftCell** address as a string in Excel format, for example, *"A1"*. In this example, *"A"* stands for the column and *"1"* for the row index.
```

Comment:
You can also specify the **topLeftCell** cell address with A1 notation. A1 notation uses the standard Excel alphanumeric sequence of column letter and 1-based row, for example, *"A1"* refers to the intersection of column "A" with row "1".
 
Note: Cell address in A1 notation must be in uppercase, like in Excel.

Reply:
[200c07d](https://github.com/DevExpress/devextreme-documentation/pull/462/commits/200c07d263984ac297da5ae0b1c5d72a2584c2ef)